TypeCobol/Compiler/Sql/CodeElements/Statements/TruncateStatement.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.IO;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Sql.Model;
+
+namespace TypeCobol.Compiler.Sql.CodeElements.Statements
+{
+    /// <summary>
+    /// SQL Truncate Statement Code Element.
+    /// </summary>
+    public class TruncateStatement : SqlStatementElement
+    {
+        public StorageArea TableName { get; }
```

Comment:
I'm not sure about `StorageArea`. We need to discuss how StorageArea are integrated in SQL model. You used IntrinsicStorageArea but it is a Cobol concept (it could be reused but we haven't decided yet).
So for now, please use TableViewCorrelationName class instead.